inbm-lib/inbm_common_lib/shell_runner.py

```diff
@@ -89,6 +90,7 @@ def run_with_log_path(self,
         @param cmd: Shell cmd to execute
         @param log_path: string format of log file's absolute path
         @param cwd: if not None, run process from this working directory
+        @param password: if password provided, it will be passed as stdin input
```

Comment:
this would be better named as 'stdin' and refactor to have any callers that need stdin pass it explicitly; e.g., when running AFULNX_64, pass in the 'yes\n', when running oras, pass it as 'stdin'.

Reply:
Sounds good to me. Will make the changes.